src/main/java/forestry/core/fluids/Fluids.java

```diff
@@ -22,56 +22,25 @@
 import net.minecraft.block.Block;
 import net.minecraft.block.Blocks;
 import net.minecraft.client.Minecraft;
+import net.minecraft.fluid.Fluid;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
 import net.minecraft.resources.IResourceManager;
 import net.minecraft.util.ResourceLocation;
 
 import net.minecraftforge.api.distmarker.Dist;
-import net.minecraftforge.fluids.Fluid;
 import net.minecraftforge.fluids.FluidStack;
 
 import net.minecraftforge.fml.loading.FMLEnvironment;
 
 import forestry.core.items.DrinkProperties;
 import forestry.core.render.ForestryResource;
 
-//TODO - fluids
 public enum Fluids {
-
-	BIO_ETHANOL(new Color(255, 111, 0), 790, 1000) {
-		@Override
-		public Block makeBlock() {
-			//			return new BlockForestryFluid(this, 300, true);
-			return Blocks.DIRT;
-		}
-	},
-	BIOMASS(new Color(100, 132, 41), 400, 6560) {
-		@Override
-		public Block makeBlock() {
-			return Blocks.DIRT;
-			//			return new BlockForestryFluid(this, 100, true);
-		}
-	},
-	GLASS(new Color(164, 164, 164), 2400, 10000) {
-		@Override
-		public int getTemperature() {
-			return 1200;
-		}
-
-		@Override
-		public Block makeBlock() {
-			return Blocks.DIRT;
-			//			return new BlockForestryFluid(this, 0, true);
-		}
-	},
+	BIO_ETHANOL(new Color(255, 111, 0), 790, 1000, 300),
+	BIOMASS(new Color(100, 132, 41), 400, 6560, 100),
+	GLASS(new Color(164, 164, 164), 2400, 10000, 0),
 	FOR_HONEY(new Color(255, 196, 35), 1420, 75600) {
```

Comment:
Now that fluids have a properly namespaced registry can we rename this back to `HONEY`? (IIRC the name was changed in the first place due to a conflict with another mod)